src/NextContainer/NextContainer.types.ts

```diff
@@ -41,6 +41,8 @@ export type NxTopology = {
   height: number;
   width: number;
   identityKey: string;
+  _layoutType: string;
```

Comment:
is there a layoutType() function to retrieve this variable? In general, we shouldn't define types for the "_" variables as they're meant to be private